src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+document.querySelector('.total-population')
+  .innerHTML = String([...document.querySelectorAll('.population')]
+    .map(pop => pop.textContent.split(',').join(''))
+    .reduce((sum, pop) => sum + +pop, 0)).split('')
```

Comment:
```suggestion
    .reduce((sum, pop) => sum + Number(pop), 0)).split('')
```
this way, imho, it's more readable